arrow-udf-wasm/src/lib.rs

```diff
@@ -146,7 +145,33 @@ impl Runtime {
         self.abi_version
     }
 
-    /// Given a function signature that inlines struct types, find the function name.
+    /// Find a function by name, argument types and return type.
+    /// The argument types and return type should be the arrow type names.
+    /// The returned [`Function`] can be used to call the function.
+    pub fn find_function(
+        &self,
+        name: &str,
+        arg_type_names: &[impl AsRef<str>],
+        return_type_name: &str,
+        is_table_function: bool,
+    ) -> Result<Function> {
```

Comment:
Can we remove `is_table_function` and add another method `find_table_function`?

Can this function accept `DataType`s rather than strings like `add_function` does?

Reply:
> Can this function accept `DataType`s rather than strings like `add_function` does?

According to https://github.com/arrow-udf/arrow-udf/blob/8a777e699a3c32adebedee8a9e3b8217f495f278/arrow-udf-macros/src/types.rs#L36, seems we cannot know whether an `arrow_schema::DataType::Utf8` is `json` or `string` or `decimal` after converting RisingWave `DataType` to `arrow_schema::DataType`.

Reply:
I guess we'd better stick to the `type_name: &str` approach, at least not making things worse(

Reply:
> Can we remove `is_table_function` and add another method `find_table_function`? In the future we will have `find_aggregate`.

Done

Reply:
> Python [add_function](https://docs.rs/arrow-udf-python/latest/arrow_udf_python/struct.Runtime.html#method.add_function) accepts `impl IntoField`, which can be a `DataType` or a `Field` with metadata for json and decimal.

Done